src/templates/error.js

```diff
@@ -1,6 +1,6 @@
-export default (id, skin = '', title = '', body = '') => {
+export default (id, title = '', body = '') => {
```

Comment:
I think we want to rename this one setup-error.js. I don't think it's necessary for us to apply the custom skin class here. There is not default so it doesn't belong in the template.